i18n/en.json

```diff
@@ -2828,6 +2840,10 @@
     "translation": "Failed to send update password email successfully"
   },
   {
+    "id": "api.user.send_user_access_token.error",
+    "translation": "Failed to send user access token added email successfully"
```

Comment:
Yeah it's a bit weird, but I just copied all the form of all the other messages and this won't go to an end user anyways